src/app/app.component.ts

```diff
@@ -84,6 +108,7 @@ export class AppComponent implements OnInit {
         this.cluster = cluster;
         this.barData = barData;
         this.singleTableData = singleTableData;
+        console.log(this.tabTitle);
```

Comment:
We probably don't want to commit the log

Reply:
This is just the "sandbox" file that we use to test the components.  It is not part of the code that gets turned into an npm package.